README.md

```diff
@@ -10,6 +10,7 @@ This is a GitHub Action wrapper around nFPM, used to produce DEBs and RPMs.
 | `arch`       | Build architecture.                      |                |
 | `version`    | Product semver version. |  |
 | `maintainer` | Maintainer name. | |
+| `vendor`     | Vendor name. | |
```

Comment:
`HashiCorp` should be in the default column here 😄

Reply:
oh yes, thank you for that catch!